tests/test_snmpuser.rb

```diff
@@ -185,7 +185,7 @@ def test_snmpuser_destroy
   end
 
   def test_snmpuser_auth_password_equal_invalid_param
-    name = 'testV3PwEqualInvalid2'
```

Comment:
Nit:  Adding an anchor to the `/invalid/` regexp should remove the need for this change